src/routes/(infoPages)/[topTierPage]/+page.svelte

```diff
@@ -1,7 +1,75 @@
 <script lang="ts">
+  import "./page.scss";
+
+  import { url as arrowIcon } from "$icons/arrow_forward";
+
+  import Button, { type Variant } from "$lib/components/Button";
+  import Card from "$lib/components/Card";
+  import ContentfulRichText from "$lib/components/ContentfulRichText";
+  import Icon from "$lib/components/Icon";
+  import VideoCard from "$lib/components/VideoCard";
+
   export let data;
-  $: ({ pageMetadata } = data);
+  $: ({ topTierPage } = data);
+  $: ({ subheading, video, description, featuredServices } = topTierPage);
+
+  const getButtonVariant = (index: number): Variant => {
+    switch (index) {
+      case 0:
+        return "primary";
+      case 1:
+        return "secondary";
+      default:
+        return "outline";
+    }
+  };
 </script>
 
-<h1>{pageMetadata?.title}</h1>
-<p>This is a stub for a Top Tier page!</p>
+{#if subheading}
+  <h2>
+    {subheading}
+  </h2>
+{/if}
+{#if description?.json}
+  <ContentfulRichText document={description.json} />
+{/if}
+{#if video?.videoUrl}
+  <VideoCard url={video.videoUrl} title={video.videoTitle} description={video.videoSubhead} />
+{/if}
+{#if featuredServices}
+  <ul class="service-group-list">
+    {#each featuredServices as item, index (item?.pageMetadata?.sys.id)}
+      <!-- TODO: Can't conditionally render a named slot, but ideally we only declare Card once here. -->
+      {#if item?.heroImage?.imageSource?.url}
+        <Card>
+          <h3 class="usa-card__heading" slot="header">{item.title}</h3>
+          <img
+            slot="image"
+            src={item.heroImage.imageSource.url}
+            alt={item.heroImage.imageSource.title}
+          />
```

Comment:
We should replace this with the `Image` component eventually, but I don't think that needs to block this PR.

Reply:
Yeah, I figure once we merge your work with the blurhashes in Redis we can do a sweep and update all the images at once.